src/components/sidenav/demoClickOutsideToClose/index.html

```diff
@@ -0,0 +1,44 @@
+<div ng-controller="AppCtrl" layout="column" style="height: 500px;" ng-cloak>
+
+    <section layout="row" flex>
+  
+      <md-sidenav class="md-sidenav-left" md-component-id="clickOutsideFalse"
+                  md-whiteframe="4" md-disable-click-key-events>
+  
+        <md-toolbar class="md-theme-indigo">
+          <h1 class="md-toolbar-tools">Disabled Click/Key to Close</h1>
+        </md-toolbar>
+  
+        <md-content layout-margin="">
+          <p>
+            This sidenav is still showing the backdrop, but users can't close the sidenav on clicking or pressing 'ESC' button.
+          </p>
+          <md-button ng-click="toggleSidenav()" class="md-accent">
+            Close this Sidenav
+          </md-button>
+        </md-content>
+  
+      </md-sidenav>
+  
+      <md-content flex layout-padding>
+  
+        <div layout="column" layout-align="top center">
+          <p>
+            Developers can also disable the click and 'Escape' button events.<br/>
```

Comment:
Let's change this to "Developers can disable closing the sidenav on backdrop clicks and 'Escape' key events."